.github/workflows/build.yml

```diff
@@ -53,5 +53,96 @@ jobs:
           key: maven-repo-${{ runner.os }}-${{ steps.get-date.outputs.date }}
 
       - name: Build with Maven
-        run: mvn -B formatter:validate verify -Dnative --file pom.xml
+        run: mvn -B formatter:validate install -Dnative --file pom.xml
 
+      - name: Kubernetes KinD Cluster
+        uses: container-tools/kind-action@v1
+        with:
+          version: v0.11.1
+          registry: true
+      - name: Install OPM and Operator SDK tool
+        uses: redhat-actions/openshift-tools-installer@v1
+        with:
+          source: "github"
+
+      - name: Install Operator Lifecycle Manager and Operator SDK into Kind
+        run: operator-sdk olm install
+
+      - name: Deploy Joke sample into Kubernetes
+        run: |
+K8S_NAMESPACE=operators
+REGISTRY_NAMESPACE=joke
+VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+OPERATOR_IMAGE=$KIND_REGISTRY/$REGISTRY_NAMESPACE/quarkus-operator-sdk-samples-joke:$VERSION
+BUNDLE_IMAGE=$KIND_REGISTRY/$REGISTRY_NAMESPACE/joke-manifest-bundle:latest
+INDEX_IMAGE=$KIND_REGISTRY/$REGISTRY_NAMESPACE/joke-manifest-index:latest
+
+# Build manifests and images
+cd samples/joke
+mvn clean package -Dquarkus.container-image.build=true -Dquarkus.container-image.registry=$KIND_REGISTRY -Dquarkus.container-image.group=$REGISTRY_NAMESPACE -Dquarkus.kubernetes.namespace=$K8S_NAMESPACE
+docker push $OPERATOR_IMAGE
+
+# Build Operator Bundle
+cd target
+opm alpha bundle generate --directory manifests --package joke --channels alpha --default alpha
+docker build -t $BUNDLE_IMAGE -f bundle.Dockerfile .
+docker push $BUNDLE_IMAGE
+
+# Install Operator via Bundle
+opm index add --bundles $BUNDLE_IMAGE --tag $INDEX_IMAGE --build-tool docker --skip-tls
+docker push $INDEX_IMAGE
+
+# Apply RBAC
+kubectl apply -f manifests/jokereconciler.csv.rbac.yml -n $K8S_NAMESPACE
+
```

Comment:
I'm curious about this actually. The Quarkus Kubernetes extension generates RBACs (and this extension enriches them). Shouldn't we use the ones that are generated instead of creating another version?

Reply:
We're not creating a new version of RBAC, we are packaging the RBAC manifests to be installed when the operator is installed.